openlibrary/components/AuthorIdentifiers.vue

```diff
@@ -29,12 +29,24 @@
 </template>
 
 <script>
+const uuidV4Validator = /^[\da-f]{8}-[\da-f]{4}-[\da-f]{4}-[\da-f]{4}-[\da-f]{12}$/
+const genericNumericValidator = /^[0-9]+$/
```

Comment:
I think both `viaf`, `goodreads`, and `librivox` all have to start with `[1-9]`, no?
```suggestion
const genericNumericValidator = /^[1-9]\d*$/
```

Reply:
Probably just because it’s simpler. :) For LibriVox specifically, [the proposal for P1899](https://www.wikidata.org/wiki/Wikidata:Property_proposal/Archive/31#P1899) doesn’t make any specific mention of the regex. It was also one of the earlier proposals (2015), so probably not a lot of thought or intention towards tighter regexes, and there wasn’t a lot of discussion about it that could have brought community highlight to it.